rules/list-role-definitions-in-acr/rule.metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "list-role-definitions-in-acr",
+    "attributes": {
+      "armoBuiltin": true
```

Comment:
Does the rule require an `actionRequired` status?

Reply:
yes, it should be "requires review" after that status is fixed